packages/blitz/src/cli/utils/next-utils.ts

```diff
@@ -224,9 +224,14 @@ export function nextBuild(
 
 export function nextExport(nextBin: string, config: ServerConfig) {
   const spawnEnv = getSpawnEnv(config)
+  let args = ["export"]
+
+  if (config.outdir) {
+    args = args.concat(["-o", `${config.outdir}`])
+  }
```

Comment:
Nitpick, but could you use array mutation instead of reassigning the original array

```suggestion
  const args = ["export"]

  if (config.outdir) {
    args.push("-o", `${config.outdir}`)
  }
```